### PR TITLE
Emoji tablet update

### PR DESCRIPTION
**Pull request by elisa-lj11**
## Test Plan
1. Load `emojiTablet.js`
2. Test that each emoji spawns and has its expected behavior
- **Corny**: should have a rim shot sound, laugh, and fall to the floor rolling
- **Flower**: when near-grabbed, should have a cute flower explosion
- **Heart**: does nothing
- **Monster**: growls, falls to the floor, and walks away from you
- **Pickle**: crunches when you eat it
- **Pizza**: when you eat it you hear "nom nom" sound and see "crumbs" (particle effects) fall out
- **Poo**: rainbow poop swirls to the floor, when you click it, it farts

### CODE REVIEW

**Comment by misslivirose** [comment]
Can you incorporate the changes in #139 into this PR to adjust for the edible bug?

File: Marketplace/emoji-tablet/behaviors/Fart.js [hunk 1]
```diff
@@ -12,22 +12,23 @@
...
+    _this = this;
+
     this.preload = function (entityID) {
         _this.entityID = entityID;
         playback = {volume: SOUND_VOLUME, position: Entities.getEntityProperties(_this.entityID, "position").position};
     };
 
     this.unload = function () {
-
+        /* nothing to put here */
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/Fart.js]: remove unused unload function

File: Marketplace/emoji-tablet/behaviors/Flower.js [hunk 2]
```diff
@@ -9,29 +9,33 @@
...
     };
 
     FlowerEmoji.prototype = {
         preload: function (entityID) {
             _this.entityID = entityID;
         },
-        unload: function () {
-
+        unload: function() {
+            /* nothing here */
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/Flower.js]: remove unused unload function

File: Marketplace/emoji-tablet/behaviors/cornyLaugh.js [hunk 3]
```diff
@@ -63,7 +76,7 @@
     }, ROLL_INTERVAL);
 
     this.unload = function() {
-
+        /* nothing here */
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/cornyLaugh.js]: remove unused unload function

File: Marketplace/emoji-tablet/behaviors/monster_growl.js [hunk 4]
```diff
@@ -14,21 +14,24 @@
...
+    };
 
     Audio.playSound(GROWL, {
         position: properties.position,
         volume: SOUND_VOLUME
     });
  
     this.unload = function() {
-        // UI and Cache cleanup etc happen here,
+        /* nothing here */
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/monster_growl.js]: remove unused unload function

File: Marketplace/emoji-tablet/behaviors/monster_walk.js [hunk 5]
```diff
@@ -39,14 +39,14 @@
...
-            "rotation": Quat.multiply(MyAvatar.orientation, {x:0, y:1, z:0, w:0}),
+            "rotation": Quat.multiply(MyAvatar.orientation, { x: 0, y: 1, z: 0, w: 0 }),
             "velocity": destination
         };
         Entities.editEntity(_entityID, newProperties);
     }, INTERVAL);
     
     this.unload = function() {
-
+        /* nothing here */
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/monster_walk.js]: remove unused unload function

File: Marketplace/emoji-tablet/behaviors/Edible.js [hunk 6]
```diff
@@ -12,113 +12,109 @@
...
+            polarFinish: 88.00000017209337,
+            azimuthStart: -180.00000500895632,
+            azimuthFinish: 180.00000500895632
         };
         Entities.addEntity(crumbsProperties1);
         Entities.addEntity(crumbsProperties2);
         Audio.playSound(NOM, {
-            position: Entities.getEntityProperties(_this.entityID, "position").position,
+            position: Entities.getEntityProperties(_this.entityID, position).position,
             volume: 0.5
```
Comment (RebeccaStankus) [inline Marketplace/emoji-tablet/behaviors/Edible.js]: magic number for volume

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
**Testing complete:**

Load emojiTablet.js **Check**
Test that each emoji spawns and has its expected behavior **Check**
Corny: should have a rim shot sound, laugh, and fall to the floor rolling **No audio is heard**
Flower: when near-grabbed, should have a cute flower explosion **Receiving a crash everytime I near-grab.**
Heart: does nothing **Check**
Monster: growls, falls to the floor, and walks away from you
Pickle: crunches when you eat it **Crashes when trying to consume / bringing up to face**
Pizza: when you eat it you hear "nom nom" sound and see "crumbs" (particle effects) fall out **Crashes when trying to consume / bringing up to face**
Poo: rainbow poop swirls to the floor, when you click it, it farts **Check**


**Fail** - beyond the crashing mentioned above, when using the app in general random crashes seem to occur even when items are spawned in, mainly Pickles & Pizzas by the way it seems.

**Pull request closed without merging**